app/src/db/migrations/20200602090631_010-forestrysectoroperatorscreening-form.js

```diff
@@ -0,0 +1,118 @@
+      table.increments('contactId').primary();
+      table.uuid('submissionId').references('submissionId').inTable(`${PREFIX}_submission`).notNullable().index();
+      table.enu('contactType', ['PRIMARY', 'COVID_COORDINATOR']).notNullable();
+      table.string('firstName').notNullable();
+      table.string('lastName').notNullable();
+      table.string('phone1', 30).nullable();
+      table.string('phone2', 30).nullable();
+      table.string('email').nullable();
+      stamps(knex, table);
+    }))
+    .then(() => knex.schema.createTable(`${PREFIX}_submission_business`, table => {
+      table.increments('businessId').primary();
+      table.uuid('submissionId').references('submissionId').inTable(`${PREFIX}_submission`).notNullable().index();
+      table.string('name').notNullable();
+      table.string('orgBookId');
+      table.string('addressLine1').notNullable();
+      table.string('addressLine2');
+      table.string('city').notNullable();
+      table.string('province', 30).notNullable();
+      table.string('postalCode', 30).notNullable();
+      stamps(knex, table);
+    }))
+    .then(() => knex.schema.createTable(`${PREFIX}_submission_attestation`, table => {
+      table.uuid('attestationId').primary();
+      table.uuid('submissionId').references('submissionId').inTable(`${PREFIX}_submission`).notNullable().index();
+
+      table.boolean('guidelinesRead').notNullable();
+      table.boolean('assessmentCompleted').notNullable();
+      table.boolean('developedPlan').notNullable();
+      table.boolean('protectionSignage').notNullable();
+      table.boolean('workerContactPersonnel').notNullable();
+      table.boolean('commonAreaDistancing').notNullable();
+
+      table.enu('sleepingAreaType', ['SINGLE', 'SHARED']).notNullable();
+      table.integer('sharedSleepingPerRoom').notNullable();
+
+      table.boolean('sharedSleepingDistancing').notNullable();
+      table.boolean('selfIsolateUnderstood').notNullable();
+      table.boolean('selfIsolateAccommodation').notNullable();
+      table.boolean('laundryServices').notNullable();
+      table.boolean('wasteManagementGloves').notNullable();
+      table.boolean('wasteManagementSchedule').notNullable();
+      table.boolean('wasteManagementBags').notNullable();
+      table.boolean('handWashingStations').notNullable();
+      table.boolean('handWashingSoapWater').notNullable();
+      table.boolean('handWashingWaterless').notNullable();
+      table.boolean('handWashingPaperTowels').notNullable();
+      table.boolean('handWashingSignage').notNullable();
+      table.boolean('distancingMaintained').notNullable();
+      table.boolean('distancingFaceShields').notNullable();
+      table.boolean('disinfectingSchedule').notNullable();
+      table.boolean('educationSignage').notNullable();
+      table.boolean('educationContactPersonnel').notNullable();
+      table.boolean('transportationSingleOccupant').notNullable();
```

Comment:
might as well leave this transport stuff in the model even if its not currently required for forestry form